core/next.config.mjs

```diff
@@ -52,15 +52,22 @@ const nextConfig = {
 	],
 	experimental: {
 		optimizePackageImports: ["@icons-pack/react-simple-icons", "lucide-react"],
+		webpackBuildWorker: true,
+		parallelServerBuildTraces: true,
 	},
 	// open telemetry cries a lot during build, don't think it's serious
 	// https://github.com/open-telemetry/opentelemetry-js/issues/4173
-	// webpack: (config, { buildId, dev, isServer, defaultLoaders, nextRuntime, webpack }) => {
-	// 	if (isServer) {
-	// 		config.ignoreWarnings = [{ module: /opentelemetry/ }];
-	// 	}
-	// 	return config;
-	// },
+	webpack: (config, { buildId, dev, isServer, defaultLoaders, nextRuntime, webpack }) => {
+		if (config.cache && !dev) {
+			config.cache = Object.freeze({
+				type: "memory",
+			});
+		}
```

Comment:
as mentioned in the pr, we could theoretically add `&& env.CI` here to only have this happen on CI, but i prefer shorter initial build times locally as well